templates/download/raspberry-pi/index.html

```diff
@@ -96,6 +93,21 @@ <h3 class="p-heading--four u-no-margin--bottom">Raspberry Pi 4</h3>
           ) | safe
         }}
       </div>
+    </div>
+  </div>
+  <div class="row u-vertically-center u-sv3">
+    <div class="col-3">
+    </div>
```

Comment:
Can you remove this empty div and use [the empty columns](https://vanillaframework.io/docs/patterns/grid#empty-columns) pattern for pushing grid columns.